lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -131,8 +246,18 @@ def self.update_xc_config(file_path, new_version, new_version_short)
           end
         end
 
-        private 
+        #----------------------------------------
+        private
+        #----------------------------------------
+
   
+        # Split a version string into its 4 parts, ensuring its parts count is valid
+        #
+        # @param [String] version The version string to split into parts
+        # @return [Array<String>] An array of exactly 4 elements, containing each part of the version string.
+        # @note If the original version string contains less than 4 parts, the returned array is filled with zeros at the end to always contain 4 items.
```

Comment:
This answered my question about the line above as soon as I thought of it. Nice!